bridges/darwinia-ethereum/bridge/Cargo.toml

```diff
@@ -34,7 +34,7 @@ postage     = "0.4"
 array-bytes = "1.4"
 serde       = { version = "1", features = ["derive"] }
 serde_json  = "1"
-codec       = { package = "parity-scale-codec", version = "3.1.0" }
+codec       = { package = "parity-scale-codec", version = "2" }
```

Comment:
why downgrade the version here?

Reply:
Yes, this pull request includes multiples not related to pangolin-parachain-rococo bridge. and this change (downgrade codec version.)
Have two reasons.

1. Currently, bridge pangolin-parachain-rococo header relay use subql+subxt and message relay use solution by parity-bridges-common. so we need to choose a suitable subxt version that is compatible with upstream.
2. Because of the bridge pangolin-ropsten also use subxt. but before this pull request the bridge pangolin-ropsten use the latest subxt. so set version to `3.1.0` now we must downgrade this version to solve cargo dependencies issues (also include change some subxt api).

Other not related changes. like move `client-pangoro` to `assistants` just to keep the code structure uniform